roles/foreman_repositories/tasks/debian.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: 'Install GPG package'
+  package:
+    name: 'gpg'
+    state: present
+  tags:
+    - packages
+  when: ansible_distribution_release == "buster"
```

Comment:
I always wondered why this is limited to buster? Same should be needed on Ubuntu too?